content/tutorials/getusermedia/intro/en/index.html

```diff
@@ -170,144 +165,147 @@ <h3 id="toc-round2">Round 2: device element</h3>
-<p>You should now use the <a href="https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices/getUserMedia" title="Mozilla Developer Network documentation for MediaDevices.getUserMedia()"><code>navigator.MediaDevices.getUserMedia()</code></a> method, which is <a href="https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices/getUserMedia#Browser_compatibility" title="MDN browser compatibility chart for MediaDevices.getUserMedia()">widely supported</a>.</p>
+<p>You should now use the <a href="https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices/getUserMedia" title="Mozilla Developer Network documentation for MediaDevices.getUserMedia()"><code>navigator.mediaDevices.getUserMedia()</code></a> method, which is <a href="https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices/getUserMedia#Browser_compatibility" title="MDN browser compatibility chart for MediaDevices.getUserMedia()">widely supported</a>.</p>
 
-<h2 id="toc-gettingstarted">Getting started</h2>
+<h2 id="toc-gettingstarted">Get started</h2>
 
-<p>With <code>getUserMedia()</code>, we can finally tap into webcam and microphone input without a plugin.
+<p>With <code>getUserMedia()</code>, you can finally tap into webcam and microphone input without a plugin.
 Camera access is now a call away, not an install away. It's baked directly into the browser. Excited yet?</p>
 
 <h3 id="toc-featuredecting">Feature detection</h3>
 
-<p>Feature detecting is a simple check for the existence of <code>navigator.mediaDevices.getUserMedia</code>:</p>
+<p>Feature detection is a simple check for the existence of <code>navigator.mediaDevices.getUserMedia</code>:</p>
 
 <pre class="prettyprint"><code>function hasGetUserMedia() {
-  return !!(navigator.mediaDevices &&
-    navigator.mediaDevices.getUserMedia);
+  return !!(navigator.mediaDevices && navigator.mediaDevices.getUserMedia);
 }
-
 if (hasGetUserMedia()) {
   // Good to go!
 } else {
-  alert('getUserMedia() is not supported by your browser');
+  alert("getUserMedia() is not supported by your browser");
 }
 </code></pre>
 
-<h3 id="toc-acccess">Gaining access to an input device</h3>
+<h3 id="toc-acccess">Gain access to an input device</h3>
 
-<p>To use the webcam or microphone, we need to request permission.
+<p>To use the webcam or microphone, you need to request permission.
 The parameter to <code>getUserMedia()</code> is an object specifying the details and
 requirements for each type of media you want to access. For example, if you want to access the webcam, the parameter should be <code>{video: true}</code>. To use both the microphone and camera,
 pass <code>{video: true, audio: true}</code>:</p>
-<pre class="prettyprint"><code>&lt;video autoplay&gt;&lt;/video&gt;
```

Comment:
"<video autoplay><script>" is missing. Just replace JS code

Reply:
Where exactly does the script tag go? Can you provide the full code with the script tags included so that I can see?

Reply:
I meant, please update only the JavaScript code, not the HTML code.

Reply:
```
<pre class="prettyprint"><code>&lt;video autoplay&gt;&lt;/video&gt;

const constraints = {
  video: true,
};
...
```

Reply:
Done.